File: led.c
#include <wiringPi.h> 

// GPIO 17, WiringPi 0
const int ledPin = 0; 

int main(void)
{
    // Inicializar pin
    wiringPiSetupGpio();
    // Definirlo como salida
    pinMode(ledPin, OUTPUT);

    while(1)
    {
	    digitalWrite(ledPin, HIGH); // LED ON
	    digitalWrite(ledPin, LOW); // LED OFF
    }

    return 0;
}

File: led.py
import RPi.GPIO as GPIO
import time

# GPIO 17
ledPin = 17 

# GPIO BCM
GPIO.setmode(GPIO.BCM) 
# Definirlo como salida
GPIO.setup(ledPin, GPIO.OUT)

# Led OFF - estado inicial
GPIO.output(ledPin, GPIO.LOW)

try:
	while 1:
	    GPIO.output(ledPin, GPIO.HIGH) # Led ON
	    GPIO.output(ledPin, GPIO.LOW) # Led OFF
except KeyboardInterrupt:
    GPIO.cleanup() # Limpiar todos GPIO
